recipes/makefun/LICENSE

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
```

Comment:
The license file is already in the source distribution so you can remove the manually packaged copy.

Reply:
Ok, I'll remove the file but leave the file name in meta.yaml

Reply:
(done)

Reply:
Yep that's correct